webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/EditableAction.scss

```diff
@@ -0,0 +1,7 @@
+.hideElement {
```

Comment:
don't we have a class for this in core already?

Reply:
Looks like I have poor grepping skills, where is it hidden?

Reply:
it's comming from bootstrap `.hide`.
Tho it might not be smart to rely on it, as core is not abusing it too much, so it might easily go away.